app/Offer.php

```diff
@@ -23,6 +23,6 @@ public static function offers(): Collection
 
     function product(): BelongsTo
     {
-        return $this->belongsTo(Product::class, 'product_id')->select('id', 'name');
+        return $this->belongsTo(Product::class, "product_id");
```

Comment:
Verandert dit niks aan de kassa kant? Bij het toevoegen van een aanbieding?

Reply:
Dit was een merge conflict die ik via de web editor heb opgelost. Ik verwacht persoonlijk dat dit geen problemen oplevert aangezien je met de bovenstaande code volgens mij exact op dezelfde manier te werk gaat als met de onderstaande code. Ik kan het alleen niet testen, want ik krijg nog steeds een navigation duplication exception wanneer ik probeer in te loggen op de kassa.